packages/core/src/TagsInput/TagsInput.js

```diff
@@ -483,7 +483,7 @@ const HvTagsInput = (props) => {
           })}
         {!(disabled || readOnly) && (
           <HvListItem
-            className={clsx(!multiline && classes.singleLine)}
+            className={clsx(!multiline && classes.singleLine, value.length === 0 ? "empty" : "")}
```

Comment:
Maybe in here we could use a conditional class instead of a conditional string something like `classes.empty`, and in the styles refer to it as `$empty` which would refer to the property/class that was defined